includes/admin/meta-boxes/views/html-variation-admin.php

```diff
@@ -48,6 +48,16 @@
 		?>
 		<input type="hidden" name="variable_post_id[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( $variation_id ); ?>" />
 		<input type="hidden" class="variation_menu_order" name="variation_menu_order[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( $variation_object->get_menu_order( 'edit' ) ); ?>" />
+		
```

Comment:
There's a hidden whitespace error here, empty lines should not contain any characters.

Reply:
Resolved in a9afc72